### PR TITLE
[ci] Migrate webui::package_helper test to rspec

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

File: src/api/spec/helpers/webui/package_helper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+RSpec.describe Webui::PackageHelper, type: :helper do
+  describe '#nbsp' do
+    it 'produces a SafeBuffer' do
+      sanitized_string = nbsp("a")
+      expect(sanitized_string).to be_a(ActiveSupport::SafeBuffer)
+    end
+
+    it 'escapes html' do
+      sanitized_string = nbsp('<b>unsafe<b/>')
```
Comment (ChrisBr) [inline src/api/spec/helpers/webui/package_helper_spec.rb]: Why the variable?
```
  expect(nbsp('<b>unsafe<b/>')).to eq('&lt;b&gt;unsafe&lt;b/&gt;') 
```

**Comment by ChrisBr** [comment]
LGTM! 

Nitpicks are the variables, but just a personal preference ...

File: src/api/spec/helpers/webui/package_helper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    it 'converts space to nbsp' do
+      sanitized_string = nbsp("my file")
+      expect(sanitized_string).to eq('my&nbsp;file')
+    end
+
+    it 'breaks up long strings' do
+      long_string = "a"*50 + "b"*50 + "c"*10
+      sanitized_string = nbsp(long_string)
+      assert_equal long_string.scan(/.{1,50}/).join("<wbr>"), sanitized_string
```
Comment (bgeuken) [inline src/api/spec/helpers/webui/package_helper_spec.rb]: Please use expect, like for your other tests:)

File: src/api/spec/helpers/webui/package_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  end
+  describe '#human_readable_fsize' do
+    skip
+  end
+  describe '#guess_code_class' do
+    skip
+  end
+  describe '#package_bread_crumb' do
+    skip
+  end
```
Comment (bgeuken) [inline src/api/spec/helpers/webui/package_helper_spec.rb]: newlines pls

**Comment by hennevogel** [comment]
Can we merge this?

**Comment by mdeniz** [comment]
LGTM :+1:

**Pull request merged**